rocAL/rocAL/include/api/rocal_api_info.h

```diff
@@ -23,53 +23,17 @@ THE SOFTWARE.
 #ifndef MIVISIONX_ROCAL_API_INFO_H
 #define MIVISIONX_ROCAL_API_INFO_H
 #include "rocal_api_types.h"
-///
-/// \param image
-/// \return The width of the ROCAL's output image in pixels
-extern "C" int ROCAL_API_CALL rocalGetOutputWidth(RocalContext rocal_context);
```

Comment:
Please check if we can retain this API's as discussed in yesterday's call

Reply:
Done